js/ui/tooltips/number-scrubber.js

```diff
@@ -79,6 +79,14 @@ TooltipEngine.classes.numberScrubber = TooltipBase.extend({
                     self.$el.addClass("dragging");
                     $(this).css("visibility", "hidden");
                     self.trigger("scrubbingStarted");
+                    // The text-to-be-tweaked needs to be the same length at the start and end
+                    // of the anti-undo changes.
+                    // I could probably just remember the length, but I like putting back the
+                    // original string. (It might even matter for i18n.)
+                    var Range = ace.require("ace/range").Range;
+                    var loc = self.aceLocation;
+                    var range = new Range(loc.row, loc.start, loc.row, loc.start + loc.length);
+                    self.originalString = parent.editor.getTextRange(range);
```

Comment:
Agh, sorry. I couldn't get gulp to work (I don't have jQuery files installed, etc. - I found the link to onboarding install info but didn't have permission to access it) so I was hacking up build/js/live-editor.tooltips.js and then trying (imperfectly) to keep it in sync with number-scrubber.js etc. So I did test it, and it worked, and then I didn't copy it back. (I had editor.session rather than editor.getSession() - I'll fix that.)